vue-thacer/src/components/TheCeramic.vue

```diff
@@ -124,10 +124,10 @@ export default {
   methods: {
     findCeramObjectProperties(id, ceramGeojson) {
       const features = ceramGeojson['features']
-      console.log(
-        //See if it could be better like this
-        ceramGeojson['features'].filter((feature) => feature.properties.ID == id)[0].properties
-      )
+
+      //See if it could be better like this
+      // ceramGeojson['features'].filter((feature) => feature.properties.ID == id)[0].properties
```

Comment:
You may remove the comment, unless you're sure it will be taken care of and/or say when why. which... is the case ? :)